x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table_columns.tsx

```diff
@@ -41,59 +42,63 @@ export const HostsTableColumns: Array<EuiBasicTableColumn<HostNodeRow>> = [
       defaultMessage: 'Operating System',
     }),
     field: 'os',
+    sortable: true,
     render: (os: string) => <EuiText size="s">{os ?? '-'}</EuiText>,
```

Comment:
Maybe we don't need this `os ?? '-'` anymore because we're handling null values in `use_host_tables`?

Reply:
yes, true, forgot it after the change. Removed 👍